dotnet/src/Agents/Abstractions/Extensions/AgentChatExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Agents.Extensions;
+
+/// <summary>
+/// Extension methods for <see cref="KernelAgent"/>
+/// </summary>
+public static class AgentChatExtensions
+{
+    /// <summary>
+    /// Add user message to chat history
+    /// </summary>
+    /// <param name="chat">The target chat.</param>
+    /// <param name="input">Optional user input.</param>
+    public static ChatMessageContent? AppendUserMessageToHistory(this AgentChat chat, string? input)
```

Comment:
Rename to `AddUserMessage` and add directly to the `AgentChat` class. Adding extension methods in the same package doesn't make sense.